src/content/updates/migration/developers.mdx

```diff
@@ -0,0 +1,112 @@
+---
```

Comment:
Developer Job to be done: set up my dev environment to use v10 components, resources, and styles to create new code artifacts or update existing code artifacts.
Questions: can v9 components and v10 components co-exist in my env? What are the differences? what are the steps to change my setup

Reply:
This one may be a little too technical for me to answer with authority. @mattrosno Do you have thoughts on this?

Reply:
On it.

Reply:
Developer environment setup belongs in getting started documentation. That doesn't need migrated; it has not changed from v9 to v10.

Updating npm packages does not need called out. That's common developer knowledge. But if it does need called out, since that's unique to each repo, it can live in each repo's migration documentation.

Reply:
"No mixy matchy" info has been added to developer section.

Reply:
Sorry @mattrosno I meant:
>Developer Job to be done: as a Carbon v9 existing developer, set up my dev environment to use v10 components, resources, and styles to create new code artifacts or update existing code artifacts.
>Questions: can v9 components and v10 components co-exist in my env? What are the differences? what are the steps to change my setup? What things are new/replaced/deprecated/removed? are there different underlying package requirements (ie React v, etc)

Reply:
@chrisconnors-ibm developer JTBDs: that information belongs https://github.com/IBM/carbon-components/blob/master/docs/migration/migrate-to-10.x.md and https://github.com/IBM/carbon-components-react/blob/master/docs/migration/migrate-to-10.x.md. I'll update those to have their respective `yarn upgrade carbon-components-react` commands to do the actual bump-up in version.

> Can v9 components and v10 components co-exist in my env?

We aren't encouraging this. It goes against common convention. Technically anything is possible, but it wouldn't be clean. Unless the need is identified and verified, I recommend not documenting this.

> What are the steps to change my setup? What things are new/replaced/deprecated/removed? Are there different underlying package requirements?

Repo-specific. Included in each repo's migration document that website links to.

> Are there different underlying package requirements?

Updating via `yarn` or `npm`, as will be documented in those two links above, will handle all dependency management automatically.

I'm not aware of any changes to React library version but I'll check to make sure.